.env

```diff
@@ -39,6 +40,11 @@ POSTFIX_RELAYHOST_TLS_LEVEL=may
 POSTFIX_RELAYHOST=False
 POSTFIX_RELAYHOST_USERNAME=False
 POSTFIX_RELAYHOST_PASSWORD=False
+# Radius
+FREERADIUS_ORGANIZATION=default
+FREERADIUS_TOKEN=djangofreeradiusapitoken
```

Comment:
why are the previous two variables needed?
In openwisp-radius the token is managed at organization level in the DB (from the admin interface).

Reply:
The freeradius rest module expects the token to be hardcoded or available via the env.  The service wont start if its missing.

Reply:
Changed this part. Now, organization name needs to be passed to container which checks if the given org has radius_token set from database, if yes, it fetches it otherwise, container exits with code 1.